files/default/github_pr_comment_trigger.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)?` (defaults to
+# `minor`), merges the PR, bumps the Chef cookbook's metadata.rb, creates a
+# version tag, and pushes up.
+
+require 'git'
+require 'json'
+require 'octokit'
+require 'pp'
+
+GIT_BRANCH = 'master'
```

Comment:
I don't expect it to happen often with cookbooks, but we do ocasionally merge into branches like `stable/1.x` when master gets a major bump that isn't everywhere yet.